test/perf/environment_setup.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+import subprocess
+import time
+
+
+def build_package():
+    print('removing build directory')
+    shutil.rmtree("build", True)
+    print('removing fairlearn.egg-info')
+    shutil.rmtree("fairlearn.egg-info", True)
+    print('removing dist directory')
+    shutil.rmtree("dist", True)
+
+    widget_path = os.path.join(os.getcwd(), 'fairlearn', 'widget', 'js')
+    os.chdir(widget_path)
+    try:
+        print('running yarn install')
```

Comment:
Would it be better to convert `build-widget.ps1` so that we don't have the same code in two languages?

Reply:
Right, there was also another issue there that prevented this from working in ADO. I decided that I don't need the widget in perf tests as of right now and removed the code that does this. So there's no duplication right now, but we should think about how to do this. Maybe build-widget.ps1 can just be a pre-step so we don't need to build the wheel here?